src/openrct2/audio/Audio.cpp

```diff
@@ -311,6 +316,22 @@ void audio_stop_all_music_and_sounds()
     audio_stop_weather_sound();
 }
 
+int32_t audio_get_device_count()
+{
+    return static_cast<int32_t>(_audioDevices.size());
+}
+
+const std::string& audio_get_device_name(int32_t index)
+{
+    Guard::Assert(index >= 0 && index < audio_get_device_count());
+    return _audioDevices[index];
```

Comment:
This will still crash in a production build, because IIRC the default behavior for the guard assert is just to log